webapp/src/js/bootstrapper/index.js

```diff
@@ -87,6 +87,17 @@
 
   /* pouch db set up function */
   module.exports = (POUCHDB_OPTIONS) => {
+    utils.checkApiAccessible()
```

Comment:
It would be better to limit this to just online users so it doesn't impact offline users at all.